Sources/JSONRPC/RPCID.swift

```diff
@@ -1,6 +1,6 @@
 import Commons
 
-public typealias RPCID = Either<String, Int>
+public typealias RPCID = Either<String, Int64>
```

Comment:
I had a question when implementing this: Is there any possibility that the SDK is going to run in a 32-bit system?